src/plonk/prover.rs

```diff
@@ -451,6 +529,17 @@ impl<C: CurveAffine> Proof<C> {
                 );
             }
 
+            for (query_index, &(wire, ref at)) in meta.aux_queries.iter().enumerate() {
+                let point_index = (*meta.rotations.get(at).unwrap()).0;
+
+                accumulate(
+                    point_index,
+                    &aux_polys[wire.0],
+                    Blind::default(),
```

Comment:
This will return Blind(1), we really want Blind(0) I think because auxiliary commitments won't have blinding factors.

Reply:
TODO: open issue to revisit this

Reply:
https://github.com/zcash/halo2/issues/25